luarules/gadgets/game_autocolors.lua

```diff
@@ -673,46 +707,64 @@ else	-- UNSYNCED
 
 		local next_team_brightness_offset = 1.0
 		local next_opponent_brightness_offset = 1.0
-		local dimming_factor = math.min( math.max(0.4, 0.05*#teamList), 0.95)
+		local dimming_factor = math.min(math.max(0.4, 0.05 * #teamList), 0.95)
 
 		for i = 1, #teamList do
 			local teamID = teamList[i]
 			local r = 1
 			local g = 1
 			local b = 1
 			if teamColorsTable[teamID] then
-				r = teamColorsTable[teamID].r/255
-				g = teamColorsTable[teamID].g/255
-				b = teamColorsTable[teamID].b/255
+				r = teamColorsTable[teamID].r / 255
+				g = teamColorsTable[teamID].g / 255
+				b = teamColorsTable[teamID].b / 255
 			end
 
 			if iconDevModeColor then
-				Spring.SetTeamColor(teamID, hex2RGB(iconDevModeColor)[1]/255, hex2RGB(iconDevModeColor)[2]/255, hex2RGB(iconDevModeColor)[3]/255)
-			elseif Spring.GetConfigInt("SimpleTeamColors", 0) == 1 or (anonymousMode == "allred" and not Spring.GetSpectatingState()) then
+				Spring.SetTeamColor(
+					teamID,
+					hex2RGB(iconDevModeColor)[1] / 255,
+					hex2RGB(iconDevModeColor)[2] / 255,
+					hex2RGB(iconDevModeColor)[3] / 255
+				)
+			elseif
+				Spring.GetConfigInt("SimpleTeamColors", 0) == 1 or (anonymousMode == "allred" and not mySpecState)
```

Comment:
The change to `mySpecState` is also a not-linting change here, but it would take `iconDevModeColor` being enabled - we can probably neglect that.